Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -5033,36 +5033,38 @@ def return_results(results):
         demisto.results(None)
         return
 
-    if results and isinstance(results, list) and len(results) > 0 and isinstance(results[0], CommandResults):
+    elif results and isinstance(results, list) and len(results) > 0:
+        result_list = []
         for result in results:
-            demisto.results(result.to_context())
-        return
+            if isinstance(result, (dict, str)):
```

Comment:
why can't we run `return_results` for them too? performance-wise?
if so, we should include `int` as well (transformer scripts)

Reply:
No because of BC. Until now if you called `return_results([entry1, entry2])`, demisto.results would've been called once with a list argument, if you will call `return_results` on each item you will be calling demisto.results for each entry (twice).